oeps/oep-0001.rst

```diff
@@ -45,163 +61,211 @@ OEP Types
 
 Finally, the Arbiter is responsible for the decision to transfer an OEP if the
-original Author has become unresponsive (as described in `Transferring OEP
+original Authors have become unresponsive (as described in `Transferring OEP
 Ownership`_).
 
-Before Submitting an OEP
-------------------------
+Architecture Team
+-----------------
 
-The OEP process begins with a new idea for Open edX. It is highly recommended
-that a single OEP contain a single key proposal or new idea. Small enhancements
-or patches often don't need an OEP and can be injected into the Open edX
-development workflow with a patch submission.
+The `edX Architecture Team`_ is accountable for assigning Arbiters to incoming
+Draft OEPs and revived OEPs that need a new Arbiter (if the original Arbiter is no
+longer available). The team can also be a resource to help/advise the Arbiter with
+the OEP process.
 
-The OEP Author should first attempt to ascertain whether the idea is appropriate 
-for an OEP.  Posting to the `edx-code`_ mailing list is the best way to go about 
-this.
+.. _edX Architecture Team: https://openedx.atlassian.net/wiki/spaces/AC/pages/439353453/Architecture+Team
+
+OEP Workflow
+============
+
+.. contents::
+  :local:
+  :depth: 2
+
+Submitting an OEP
+-----------------
+
+Step 1. Scope your idea
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The OEP process begins with a new idea for Open edX. Unless it is a Best
+Practice, a single OEP should contain a single key proposal or new idea. Small
+enhancements or patches often don't need an OEP and can be injected into the Open
+edX development workflow with a patch submission.
+
+Step 2. Vet your idea
+~~~~~~~~~~~~~~~~~~~~~
+
+OEP Authors should first attempt to ascertain whether the idea is appropriate 
+for an OEP. Posting to the `open-edx-proposals Slack channel`_ is the best way
+to go about this.
 
 Vetting an idea publicly before going as far as writing an OEP is meant to save
-the potential author time. Many ideas have been brought forward for changing
-Open edX that have been rejected for various reasons. Asking the Open edX
-community first if an idea is original helps prevent too much time being spent
-on something that is guaranteed to be rejected based on prior discussions
-(searching the Internet is not always sufficient). It also helps to make
-sure the idea is applicable to the entire community and not just the author.
-Just because an idea sounds good to the author does not mean it will work for
-most Open edX installations.
-
-Initial Submission
-------------------
-
-Once the Author has asked the Open edX community whether an idea has any chance
+the potential authors time. Asking the Open edX community first if an idea is 
+original and appropriate helps prevent wasted effort. It also helps to make
+sure the idea is applicable to the entire community and not just the authors.
+
+.. _open-edx-proposals Slack channel: https://openedx.slack.com/messages/C1L370YTZ/details/
```

Comment:
I like Slack but, between work and extracurricular activities, I have five distinct Slack accounts. They can get distracting. As someone who is less active in the community, I rely more on the mailing list to see broader changes. I agree that Slack is a great forum for the conversation; however, at least mentioning the new OEP and linking to the Slack conversation on the mailing list would help keep informed those of us who aren't always on Slack.

Reply:
P.S. Coincidentally, the mailing list is how I learned of this request for comments. 😀

Reply:
Good feedback.  I've updated the doc to include Slack as one of the options for Vetting the OEP, but keeping the edx-code mailing list as a place to announce the OEP.